parachain/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -183,13 +185,16 @@ pub mod pallet {
 		StorageMap<_, Identity, H256, CompactExecutionHeader, OptionQuery>;
 
 	#[pallet::storage]
+	#[pallet::getter(fn validators_root)]
 	pub(super) type ValidatorsRoot<T: Config> = StorageValue<_, H256, ValueQuery>;
 
 	#[pallet::storage]
+	#[pallet::getter(fn latest_finalized_header)]
 	pub(super) type LatestFinalizedHeaderState<T: Config> =
```

Comment:
would prefer we also rename to `LatestFinalizedHeader`

Reply:
Agree! Can you make the change?